test/lib/completions/ssh.exp

```diff
@@ -76,7 +76,7 @@ foreach h [get_known_hosts] {
         }
     }
 }
-assert_complete $hosts "ssh $char" $test -ltrim-colon-completions -expect-cmd-minus "$char"
+assert_complete $hosts "ssh $char" $test -expect-cmd-minus "$char"
```

Comment:
ISTR the colon stuff could be related to IPv6 addresses possibly among the returned completions. Not sure about it though, but I think it's clear that `-ltrim-colon-completions` shouldn't be breaking cases where there are no colons present in the first place.

(Sigh, I'm starting to get really tired of trying to figure out and cope with the test suite code, TCL, and expect. Anyone up to rewriting the test suite in [pexpect](https://pexpect.readthedocs.io/en/stable/)?)